client/src/components/Form/Elements/FormDialog.vue

```diff
@@ -0,0 +1,68 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faFolderOpen } from "@fortawesome/free-solid-svg-icons";
+import DataDialog from "../../DataDialog/DataDialog.vue";
```

Comment:
Please use non-relative paths when importing unless the file is in the same directory, so in this case e.g. `import DataDialog from "@/components/DataDialog/DataDialog.vue"`.